pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+NOTE:
```

Comment:
This file doesn't belong to this pull request since it's not solving the issue #3. Please remove it.